scripts/GetClrDbg.ps1

```diff
@@ -0,0 +1,128 @@
+﻿<#
+.SYNOPSIS
+Downloads the given $Version of clrdbg for the given $RuntimeID and installs it to the given $InstallPath
+
+.DESCRIPTION
+The following script will generate a project.json and NuGet.config and use dotnet restore and publish to install clrdbg, the .NET Core Debugger
+
+.PARAMETER Version
+Specifies the version of clrdbg to install. Can be 'latest', VS2015U2, or a specific version string i.e. 14.0.25229-preview-2963841
+
+.PARAMETER RuntimeID
+Specifies the .NET Runtime ID of the clrdbg that will be downloaded. Example: ubuntu.14.04-x64. Defaults to the Runtime ID of the current machine.
+
+.Parameter InstallPath
+Specifies the path where clrdbg will be installed. Defaults to the directory containing this script.
+
+.INPUTS
+None. You cannot pipe inputs to GetClrDbg.
+
+.EXAMPLE
+C:\PS> .\GetClrDbg.ps1 -Version latest -RuntimeID ubuntu.14.04-x64 -InstallPath .\clrdbg
+
+.LINK
+https://github.com/Microsoft/MIEngine
+#>
+
+Param (
+    [Parameter(Mandatory=$true)]
+    [string]$Version,
```

Comment:
Mandatory applies to each ParameterSet. ParameterSets are groups of parameters, when PS parses the parameters it falls into a single ParameterSet. If it gets params from multiple sets it throws an error. This way it would force the user to specify exactly one of "-Version 14.0.23232" or "-VersionName latest". I believe this is the behavior you are looking for but I'm not 100% sure on that.